[Security Center] Cloudforce One API docs (#20520)
cloudflare/cloudflare-docs: src/content/docs/security-center/cloudforce-one/index.mdx

1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com/) and select your account.
2. Go to **Security Center** > **Threat Intelligence**.

> Cloudforce One is also available on the [API](https://developers.cloudflare.com/api/resources/cloudforce_one/subresources/requests/subresources/assets/).

Reviewer: ```suggestion
You can also use Cloudforce One via [REST API](https://developers.cloudflare.com/api/resources/cloudforce_one/subresources/requests/subresources/assets/).
```
